config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: quay.io/grafana-operator/grafana-operator
+  newTag: v5.0.0
```

Comment:
Can we jinja template ${VERSION} here instead? :thinking:

Reply:
So this is actually solved through the makefile
`IMG ?= quay.io/grafana-operator/grafana-operator:$(VERSION)`
So when `make deploy` it will generate the file for us so we don't have to think of it :).

Reply:
Ahhh, sweet! :+1: